packages/server/src/api/routes/tests/search.spec.ts

```diff
@@ -966,6 +966,149 @@ describe.each([
     })
   })
 
+  !isInternal &&
+    describe("datetime - time only", () => {
+      const T_1000 = "10:00"
+      const T_1045 = "10:45"
+      const T_1200 = "12:00"
+      const T_1530 = "15:30"
+      const T_0000 = "00:00"
+
+      const UNEXISTING_TIME = "10:01"
+
+      beforeAll(async () => {
+        await createTable({
+          time: { name: "time", type: FieldType.DATETIME, timeOnly: true },
+        })
+
+        await createRows(
+          _.shuffle([T_1000, T_1045, T_1200, T_1530, T_0000]).map(time => ({
```

Comment:
I like the shuffle here. I should do this in the rest of the tests.

Reply:
I actually changed it in order to be used everywhere